resources/views/includes/footer.blade.php

```diff
@@ -0,0 +1,3 @@
+<div id="footer" class="animate animate-up">
+    <p id="home_copyright">All images on this website are subject to Copyright by {{ $infos['owner'] }} &copy; 2019</p>
```

Comment:
Not sure about hard codingthe year here. Maybe just leave it out?

Reply:
https://www.copyrightlaws.com/copyright-symbol-notice-year/